larva/src/main/java/org/frankframework/larva/LarvaTool.java

```diff
@@ -1014,6 +1014,7 @@ private static Entry<Object, Object> rewriteClassName(Entry<Object, Object> e) {
 			return e;
 		}
 		String newClassName = e.getValue().toString().replace(ClassNameRewriter.LEGACY_PACKAGE_NAME, ClassNameRewriter.ORG_FRANKFRAMEWORK_PACKAGE_NAME);
+		newClassName = newClassName.replace(ClassNameRewriter.LEGACY_PACKAGE_NAME_LARVA, ClassNameRewriter.CURRENT_PACKAGE_NAME_LARVA);
```

Comment:
Perhaps it's a matter of taste but I think it's cleaner to append the second `.replace()` to the first instead of making a new statement.
I'd put the chain of methods on separate lines too for increased readability.

This means we do not have to re-assign the variable, keeping it effectively final.

```suggest
		final String newClassName = e.getValue()
				.toString()
				.replace(ClassNameRewriter.LEGACY_PACKAGE_NAME, ClassNameRewriter.ORG_FRANKFRAMEWORK_PACKAGE_NAME)
				.replace(ClassNameRewriter.LEGACY_PACKAGE_NAME_LARVA, ClassNameRewriter.CURRENT_PACKAGE_NAME_LARVA);
```

Reply:
Yes, sure, that's fine, no problem. Will do that tomorrow. I like that style more as well.. probably too tired of the whole day, that I didn't do that myself. I do wonder btw how much the actual compiled byte code is different in both variants.. but that's a bit out of scope 😉